app/controllers/api/v4/stripe_cards_controller.rb

```diff
@@ -27,6 +27,57 @@ def transactions
         @has_more = false # TODO: implement pagination
       end
 
+      def create
+        event = authorize(Event.find(params[:stripe_card][:event_id]))
+        authorize event, :create_stripe_card?, policy_class: EventPolicy
+
+        sc = params.require(:stripe_card).permit(
+          :event_id,
+          :card_type,
+          :stripe_shipping_name,
+          :stripe_shipping_address_city,
+          :stripe_shipping_address_line1,
+          :stripe_shipping_address_postal_code,
+          :stripe_shipping_address_line2,
+          :stripe_shipping_address_state,
+          :stripe_shipping_address_country,
+          :stripe_card_personalization_design_id,
+          :birthday
+        )
+
+        if current_user.birthday.nil?
+          user_params = sc.slice("birthday(1i)", "birthday(2i)", "birthday(3i)")
+          current_user.update(user_params)
+        end
```

Comment:
For the API, it may be better to error if birthday is unset. They should first set the birthday, then recall this endpoint to issue the card.